### DISCUSSION TITLE
How to speed up the hexo command in WSL?

### DISCUSSION THREAD

**Discussion by Daucloud** [hexojs/hexo#5493] [Question / Answer] [answered]
I found using the hexo command (no mather `hexo cl`, `hexo g`, `hexo s` even if the `hexo --version`😥) much slower in WSL than Windows cmd, which is quite annoying. Why is it much slower using WSL, and how can I speed up the hexo commands in WSL?

**Comment by yoshinorin** [comment, marked as answer]
@Daucloud 

It may be improved by either or both of the following.

## 1. Filesystem

Are you placing your Hexo project in a directory that starts with `\\wsl$\` (or `~`) ?  If you are placing it in a directory that starts with `/mnt`, the performance will degrade because files will be across different file systems.

See: https://learn.microsoft.com/en-us/windows/wsl/filesystems#file-storage-and-performance-across-file-systems

## 2. Memory

You can allocate more memory to WSL using the `.wslconf` file.

See: https://learn.microsoft.com/en-us/windows/wsl/wsl-config#example-wslconf-file

**Reply by Daucloud** [reply]
Thanks a lot!!
